pypeit/reduce.py

```diff
@@ -104,11 +102,11 @@ def get_instance(cls, sciImg, spectrograph, par, slitTrace, waveTilts, wv_calib,
         """
         return next(c for c in cls.__subclasses__()
                     if c.__name__ == (spectrograph.pypeline + 'Reduce'))(
-            sciImg, spectrograph, par, slitTrace, waveTilts, wv_calib, objtype, ir_redux=ir_redux, det=det,
+            sciImg, spectrograph, par, caliBrate, objtype, ir_redux=ir_redux, det=det,
             std_redux=std_redux, show=show,binning=binning, setup=setup)
 
 
```

Comment:
You just undid my changes from your last PR that broke 2d coadd. You can fix it yourself this time.

Reply:
This was intended, as discussed in the PypeIt meeting.